server/modules/tasks.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  addSyncMap,
+  addSyncMapFilter,
+  BaseServer,
+  ChangedAt,
+  NoConflictResolution
+} from '@logux/server'
+import { defineSyncMapActions, LoguxNotFoundError } from '@logux/actions'
+
+import {
+  changeTask,
+  createTask,
+  deleteTask,
+  findTask,
+  getUserTasks
+} from '../db.js'
+import { Task } from '../../api/index.js'
+
+const modelName = 'tasks'
+
+const [createTaskActionType] = defineSyncMapActions(modelName)
+
+export default (server: BaseServer): void => {
+  addSyncMap<Task>(server, modelName, {
+    async access(ctx, id, action) {
+      if (createTaskActionType.type === action.type && 'fields' in action) {
```

Comment:
```suggestion
      if (createTaskActionType.match(action)) {
```

It will automatically set `action` type to `tasks/created` action’s type and you will not need `'fields' in action` check